APE10.rst

```diff
@@ -63,10 +63,10 @@ Version      Planned release date     Notes
 v1.3         December 2016
 v2.0 - LTS   June 2017                Supported with bug fixes until end of 2019
 v3.0         December 2017            First release to support only Python 3+
-v3.1         June 2018
-v3.2         December 2018
-v3.3         June 2019
-v4.0 - LTS   December 2019            Supported with bug fixes for two years
+v3.1         October 2018
+v3.2         April 2019
+v3.3         October 2019
+v4.0 - LTS   April 20120              Supported with bug fixes for two years
```

Comment:
That's a lot to wait for v4.0!!! :wink:

Reply:
Darn, you caught my scheme to make it so we don't have to work so hard on support!  (thanks! 😉 )